### PR TITLE
Haiku2048

### PR DESCRIPTION
**Pull request by ohnx**
Add Haiku2048

### CODE REVIEW

File: haiku-games/2048/2048-1.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,47 @@
+SUMMARY="Clone of the game 2048"
```
Comment (humdingerb) [inline haiku-games/2048/2048-1.0.0.recipe]: Suggestion: "The tile-matching game 2048"

File: haiku-games/2048/2048-1.0.0.recipe [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+DESCRIPTION="Clone of the popular 2048 game.
+
+HOW TO PLAY: Use your arrow keys to move the tiles.
```
Comment (humdingerb) [inline haiku-games/2048/2048-1.0.0.recipe]: I'd remove the repeat of the SUMMARY and go with:
"The goal of this popular tile game is get to tile 2048.
Use the cursor keys to move the tiles. When two tiles with the same number \
touch, they merge into one."

File: haiku-games/2048/2048-1.0.0.recipe [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+When two tiles with the same number touch, they merge into one!
+
+Your goal is to get to the 2048 tile!
+"
```
Comment (humdingerb) [inline haiku-games/2048/2048-1.0.0.recipe]: The closing " should be in the same line as the last DESCRIPTION line.

File: haiku-games/2048/2048-1.0.0.recipe [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:gcc
```
Comment (humdingerb) [inline haiku-games/2048/2048-1.0.0.recipe]: Use cmd:g++ instead

**Comment by ohnx** [comment]
also i broke the VERSION="2"

**Comment by ohnx** [comment]
@humdingerb Squashed them.

**Comment by fbrosson** [comment]
Now that this PR has been merged, the branch it is based on is no longer needed.
You should therefore click "Delete branch" for this PR to get rid of an unneeded branch in your repo. Thanks!
GitHub help: [Deleting unused branches](https://help.github.com/articles/deleting-unused-branches/)

**Pull request merged**